Gruntfile.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * gpii-grunt-mdjson-lint
+ *
+ * Copyright (c) 2017 Raising the Floor Internationals
+ *
+ * Licensed under the BSD-3-Clause license.
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.require("%gpii-grunt-lint-all/src/load-npm-tasks-properly.js");
+module.exports = function (grunt) {
+    // This is required because of the inherent circularity of this setup.
+    gpii.grunt.lintAll.fixGruntTaskLoading(grunt);
+
+    grunt.initConfig({
+        lintAll: {
+            sources: {
+                md: ["./*.md", "./**/*.md", "!./node_modules/**", "!./tests/markdown/bad*"],
```

Comment:
What did we decide about the possibility of "ignores" that always bypasses top-level node_modules directories?

Reply:
I had to review to confirm.  The gpii-grunt-lint-all plugin does not use expansion or any other Fluid conventions, but uses grunt's own variable syntax to mash together options for each type of file, as shown here:

https://github.com/GPII/gpii-grunt-lint-all/blob/master/tasks/lint-all.js#L39

The best I could manage without more serious surgery is to add an ignores option, which is included in each check.  The caveat is that those ignores would have to be expressed as negated patterns.  If that's acceptable, I can write up that work against gpii-grunt-lint-all and knock that out in short order.